packages/core/src/routes/hook.ts

```diff
@@ -55,8 +96,15 @@ export default function hookRoutes<T extends AuthedRouter>(
   router.patch(
     '/hooks/:id',
     koaGuard({
-      params: z.object({ id: z.string().min(1) }),
-      body: Hooks.createGuard.omit({ id: true }).partial(),
+      params: z.object({ id: z.string() }),
```

Comment:
What if a user calls the POST and PATCH API using the old schema with the event?

e.g. POST /hooks {event: SignIn}
then PATCH /hooks/:id { event: Register }

Reply:
`POST /hooks {event: "PostSignIn"}` => `{ event: "PostSignIn", events: [] }`
`PATCH /hooks/:id { event: "PostRegister" }` => `{ event: "PostRegister", events: [] }`